src/components/navbar/navbar.component.tsx

```diff
@@ -13,6 +13,7 @@ import {
   HeaderGlobalAction
 } from "carbon-components-react/es/components/UIShell";
 import { LoggedInUser, UserSession } from "../../types";
+import styles from "./UserAvatar_styles.css";
```

Comment:
Can you stay consistent with the naming? I don't think we use an underscore for files anywhere. Also usually the CSS files are called similar to the tsx files. So it should rather be "navbar.scss" or "navbar.component.scss".